test.js

```diff
@@ -1,29 +1,72 @@
+var assert = require("assert");
 var Bezier = require("./lib");
 
-function check(b) {
-  console.log("---");
-  console.log(b.toString());
-  console.log(b.length());
-  console.log(b.dpoints);
-  console.log(b.derivative(0), b.derivative(0.5), b.derivative(1));
-  console.log(b.normal(0), b.normal(0.5), b.normal(1));
+// plain and SVG quadratic check
+[
+  new Bezier(0,0, .5,1, 1,0),
+  Bezier.fromSVG("M 0 0 Q 0.5 1 1 0")
+]
+  .forEach(function(b) {
+    assert.equal(b.toString(), "[0/0, 0.5/1, 1/0]");
+    assert.equal(b.length(), 1.4789428575453212);
```

Comment:
That feels a little too brittle, can we do a difference-epsilon computation here based on how long we know "true" maths says this should be?